.github/workflows/pr.yml

```diff
@@ -0,0 +1,14 @@
+name: "Lint PR"
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
```

Comment:
Is it trustworthy?

Reply:
can we add something like just to double-check the run only title changes:
`if: ${{ github.event.changes.title.from && github.event.changes.title.from != github.event.pull_request.title }}`

Reply:
in the context of GitHub Actions, the block types: [opened, reopened, synchronize, edited] specifies which pull_request events will trigger the workflow. Here's what each type means:

opened: The workflow is triggered when a new Pull Request is created.

reopened: The workflow runs when a previously closed Pull Request is reopened.

synchronize: The workflow runs when new commits are pushed to the branch associated with the Pull Request, such as when you add changes after opening the PR.

edited: The workflow is triggered when the Pull Request is edited. This includes changes to the title, description, or metadata.

We listening all theses events @eduardoformiga . So I think that's enough

Reply:
question: wouldn't it be enough to use `opened`, `reopened` and `edited`?

Reply:
It seems like `synchronize` isn't relevant to our case 🤔

Reply:
It makes sense. I just follow the action documentation. But I can't see a case where we can change the PR title through commits.

Reply:
I found a case just now. If I change the action code I want it to run.